hw/amdc_revd.bd

```diff
@@ -5,8 +5,7 @@
       "device": "xc7z030sbg485-1",
       "name": "amdc_revd",
       "synth_flow_mode": "Hierarchical",
-      "tool_version": "2019.1",
-      "validated": "true"

```

Comment:
@mimayberry it looks like you committed the bd file in an unvalidated state. Can you validate it and recommit it? This typically will change the bd file diff dramatically. I want to ensure we only merge validated block diagrams.

Reply:
I just validated the bd file, should be good to go @npetersen2